src/shared/components/getting-started-section/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import HexSvg from 'shared/media/backgrounds/hexagons.svg';
+import stepsArr from 'shared/data/getting-started';
+import SyntaxHighlighter from 'shared/components/syntax-highlighter';
+import Button from 'shared/components/button';
+import styles from './index.module.css';
+
+const GettingStarted = ({ intl }) => {
+    const messages = intl.messages;
+
+    const steps = stepsArr.map((step, index) =>
+        (
+            <div className={ styles.command } key={ index }>
+                <p className={ styles.title }>{ messages[step.title] }</p>
+                <p className={ styles.desc }>{ newLineToBreak(messages[step.desc]) }</p>
+                <SyntaxHighlighter codeStr={ step.codeStr } />
+                { step.note && <p className={ styles.note } >{ messages[step.note] }</p> }
+            </div>
+        )
+    );
```

Comment:
I would go for a new component for the steps here.

Reply:
Fixed.